controllers/components/datasciencepipelines/datasciencepipelines_controller.go

```diff
@@ -19,40 +19,71 @@ package datasciencepipelines
 import (
 	"context"
 
-	"k8s.io/apimachinery/pkg/runtime"
+	securityv1 "github.com/openshift/api/security/v1"
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	extv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
 
 	componentsv1 "github.com/opendatahub-io/opendatahub-operator/v2/apis/components/v1"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/controller/actions/deploy"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/controller/actions/render"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/controller/actions/render/kustomize"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/controller/actions/updatestatus"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/controller/predicates/resources"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/controller/reconciler"
+	odhdeploy "github.com/opendatahub-io/opendatahub-operator/v2/pkg/deploy"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/metadata/labels"
 )
 
-// DataSciencePipelinesReconciler reconciles a DataSciencePipelines object.
-type DataSciencePipelinesReconciler struct {
-	client.Client
-	Scheme *runtime.Scheme
-}
+var (
+	DefaultPath = odhdeploy.DefaultManifestPath + "/" + componentsv1.DataSciencePipelinesComponentName + "/base"
```

Comment:
yep, this could probably be moved in the `initialize` action

Reply:
Updated this to be a Manifest info now, and stopped exporting it. I did however keep it as part of the package scope because it is needed in the components `Init` function for the apply